server/graphql/v2/object/CurrencyFxRate.ts

```diff
@@ -0,0 +1,36 @@
+import { GraphQLBoolean, GraphQLFloat, GraphQLNonNull, GraphQLObjectType } from 'graphql';
+import { GraphQLDateTime } from 'graphql-scalars';
+
+import { Currency } from '../enum';
+import { CurrencyFxRateSourceType } from '../enum/CurrencyFxRateSourceType';
+
+const CurrencyFxRate = new GraphQLObjectType({
+  name: 'CurrencyFxRate',
+  description: 'Fields for a currency fx rate',
+  fields: () => ({
+    percentage: {
```

Comment:
Not sure about this name. Alternatives: `value`, `rate`?

Reply:
`value` sounds good, it's consistent with what we do elsewhere